{{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/test-login.cy.ts

```diff
@@ -2,12 +2,9 @@ describe('Tests login workflow', () => {
   it('Home page auto redirects to login', () => {
     cy.visit('/')
     cy.url().should('include', '/log-in')
-  })
-  it('Filling in email and passwords goes to', () => {
     cy.get('[data-cy=email]').type(Cypress.env('TEST_USER_EMAIL'))
     cy.get('[data-cy=password]').type(Cypress.env('TEST_USER_PASS'))
-    cy.get('[data-cy="login-btn"]').click()
-    cy.url().should('include', '/home')
+    cy.contains('[data-cy="login-btn"]', 'Login').click()
```

Comment:
As you now are also checking for the button content. This should be `Log in`. Right now the test is failing bc of this